site/content/blogs/release-v121.en.md

```diff
@@ -28,12 +28,53 @@ Copilot v1.21 brings several new features and improvements:
 ## CloudFront Integration
 
 ## Configure Environment Security Group
```

Comment:
What do you think of creating a section like this and moving these under the it?
```
## New Environment Manifest Features

### Configure Environment Security Group
...
### Add Support For ELB Access Logs

```